### PR TITLE
Make test_for_reproducibility uworker safe.

### PR DESCRIPTION
**Pull request by jonathanmetzman**
All tasks that take a testcase as an argument use this function.

### CODE REVIEW

**Comment by jonathanmetzman** [comment]
Follow up to finish testcase_manager.py https://github.com/google/clusterfuzz/pull/3865

File: src/clusterfuzz/_internal/protos/uworker_msg.proto [hunk 1]
```diff
@@ -114,6 +114,11 @@ message Input {
   optional string module_name = 18;
   optional google.protobuf.Timestamp preprocess_start_time = 19;
   repeated string global_blacklisted_functions = 20;
+  optional TestCaseManagerInput testcase_manager_input = 21;
+}
+
+message TestCaseManagerInput {
+  optional google.datastore.v1.Entity fuzz_target = 1;
```
Comment (oliverchang) [inline src/clusterfuzz/_internal/protos/uworker_msg.proto:121]: Would it make sense to consolidate the `fuzz_target` here with what's in FuzzTaskInput? 

e.g. either `FuzzTaskInput` would use `TestCaseManagerInput`, or we move `fuzz_target` up into `Input` ?

Comment (jonathanmetzman) [reply src/clusterfuzz/_internal/protos/uworker_msg.proto:121]: I think hoisting it into Input makes sense.

**Review by oliverchang — APPROVED** [review]
LGTM assuming we hoist `fuzz_target` up into `Input`.

**Pull request merged**